classes/mock/generator.php

```diff
@@ -668,7 +668,7 @@ protected function getReturnType(\reflectionMethod $method)
         return $returnTypeCode;
     }
 
-    protected function isNullable(\reflectionType $type)
+    protected function isNullableReturnType(\reflectionType $type)
```

Comment:
I would leave this as is because it could break extensions or any code extending this class.